### PR TITLE
Applications list for Multicluster

### PR DESCRIPTION
**Pull request by josunect**
Fixes https://github.com/kiali/kiali/issues/5961

### CODE REVIEW

**Comment by hhovsepy** [comment]
It seems to be an existing issue in UI, when selecting all Namespaces, it requests workloads and applications twice, so the data is duplicated.
Screenshot from 2023-04-05 20-20-46

**Comment by hhovsepy** [comment]
+1 in listing separate app per cluster for the duplicate namespace

File: business/apps.go [hunk 1]
```diff
@@ -109,78 +134,86 @@ func (in *AppService) GetAppList(ctx context.Context, criteria AppCriteria) (mod
...
+				if appItem.IstioSidecar = w.IstioSidecar; !appItem.IstioSidecar {
+					break
+				}
 			}
-		}
-		if criteria.IncludeHealth {
-			appItem.Health, err = in.businessLayer.Health.GetAppHealth(ctx, criteria.Namespace, appItem.Name, criteria.RateInterval, criteria.QueryTime, valueApp)
-			if err != nil {
-				log.Errorf("Error fetching Health in namespace %s for app %s: %s", criteria.Namespace, appItem.Name, err)
+			if criteria.IncludeHealth {
```
Comment (leandroberetta) [inline business/apps.go:208]: I was thinking if there is any work related to mc in the health computation, I noticed that there some queries to get the traffic rate that is used in the health, but I don't think we need to include the cluster value in the queries, but I'm not sure. I think that we need to check any inbound and outbound connection to the specific application (from the application's cluster or other clusters too).

**Review by leandroberetta — COMMENTED** [review]
Code looks good to me. I have just a question related to the health.

**Review by leandroberetta — DISMISSED** [review]
Code looks good and also verified that works fine. Regarding my health question, health for mc can be done in a follow up PR.

**Comment by josunect** [comment]
> Code looks good and also verified that works fine. Regarding my health question, health for mc can be done in a follow up PR.

Thanks, @leandroberetta ! I can create the follow up PR and linked to this one.
I have rebased.

**Review by leandroberetta — DISMISSED** [review]

**Review by leandroberetta — DISMISSED** [review]
LGTM.

**Comment by hhovsepy** [comment]
It seems the changes https://github.com/kiali/kiali/pull/5944/files#diff-d9a135c62e989128a43034ac55d64a837f21fedc602dbeadb6709874fa32ce3bR110 are somehow lost, so on single cluster case the columns are mixed, thus the test fails.

**Comment by josunect** [comment]
> It seems the changes https://github.com/kiali/kiali/pull/5944/files#diff-d9a135c62e989128a43034ac55d64a837f21fedc602dbeadb6709874fa32ce3bR110 are somehow lost, so on single cluster case the columns are mixed, thus the test fails.

Thanks, Hayk! I've just sent a commit.

**Review by hhovsepy — APPROVED** [review]
Verified:
Screenshot from 2023-04-13 18-38-08

**Pull request merged**